browser/src/canvas/sections/ShapeHandlesSection.ts

```diff
@@ -911,7 +911,16 @@ class ShapeHandlesSection extends CanvasSectionObject {
 	}
 
 	onMouseMove(position: number[], dragDistance: number[]) {
-		if (this.containerObject.isDraggingSomething() && !app.file.textCursor.visible) {
+		let canDrag = !app.file.textCursor.visible;
+
+		if (canDrag && app.map._docLayer._docType === 'presentation') {
+			// Tables get selected when multiple cells are selected. In this case, we check if DeleteRows is disabled.
+			// Because in non-edit mode, deleteRows is disabled. So we can drag the table.
+			const deleteRowsState = app.map['stateChangeHandler']._items['.uno:DeleteRows'];
```

Comment:
we use `getItemValue()`, better to not interract with internals:

https://github.com/CollaboraOnline/online/blob/01994d7dfade940835b4c00c6817f0cbabfd4423/browser/src/control/Control.PresentationBar.js#L161